jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonUnmarshaller.java

```diff
@@ -4304,7 +4305,7 @@ protected void applyStartEventProperties(StartEvent se, Map<String, String> prop
     }
 
     protected void applyMessageProperties(Message msg, Map<String, String> properties) {
-        if(properties.get("name") != null) {
+        if(properties.get("name") != null && properties.get("name").length() > 0) {
```

Comment:
Maybe predicate function will be good for this purpose something like:
`private boolean isPropertyEmpty(Map<String, String> properties, String propertyName) {
    return properties.get(propertyName) != null && properties.get(propertyName).length() > 0;
}` 
If we will use it we never will meet issues like this again. And if something will be changed we can easily modify it in one place.